### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by asifthewebguy**
fixed spelling at quiz-state.md

### CODE REVIEW

**Review by gralp-1 — CHANGES_REQUESTED** [review]
repetitive

File: hugo/content/courses/vscode-tricks/ide-tasks.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Tasks
-description: Use tasks to automatic repeitive jobs in the terminal 
+description: Use tasks to automatic repeititive jobs in the terminal
```
Comment (Piekoszowski) [inline hugo/content/courses/vscode-tricks/ide-tasks.md:3]: Correct word here is repetitive

**Review by Piekoszowski — CHANGES_REQUESTED** [review]

File: hugo/content/courses/vscode-tricks/ide-tasks.md (hunk 1 above)
Comment (asifthewebguy) [reply hugo/content/courses/vscode-tricks/ide-tasks.md]: right, I made made the mistake, while fixing the typo 😂. just pushed another fix

File: hugo/content/courses/vscode-tricks/ide-tasks.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Tasks
-description: Use tasks to automatic repeititive jobs in the terminal 
+description: Use tasks to automatic repetitive jobs in the terminal
```
Comment (Piekoszowski) [inline hugo/content/courses/vscode-tricks/ide-tasks.md:3]: looks good now

**Review by Piekoszowski — APPROVED** [review]

**Pull request closed without merging**